src/main/java/parser/ViewListCommand.java

```diff
@@ -0,0 +1,12 @@
+package main.java.Parser;
+
+public class ViewListCommand extends Command {
+
+    public ViewListCommand() {}
```

Comment:
Empty constructor may be unnecessary when there are no other constructors in the same class. It is added automatically by the compiler at compile time.